### ISSUE TITLE
Duplicated web presence

### ISSUE THREAD

**Issue by anthonydillon** [canonical/canonical-design#12]
We now have two websites representing the design team:
1. https://design.ubuntu.com/
2. https://canonical.design

This will cause confusion, out-of-date content and weaken's our SEO.

**Comment by gerwitz** [comment]
The intention is to retire [design.ubuntu.com](https://design.ubuntu.com/), but we will need to do an audit to ensure we're not decommissioning anything important and perhaps set up some redirects.

@unco I think we involve Kasia for this, unless you're confident we already reconciled this?

**Comment by unco** [comment]
Yes but more concerned having the right redirects in place. Where possible
for specific pages like the brand.

**Comment by juanruitina** [comment]
There's also https://design.canonical.com which points to the blog, we should arguably redirect it too.

**Comment by diana-stanciulescu** [comment]
@mtruj013 there is work in progress to replace all the content on design.ubuntu with new, appropriate content this cycle, host it on ubuntu.com/design and potentially adding a redirect or completely sunsetting design.ubuntu (tbd). 

epic on the brand board: https://warthogs.atlassian.net/browse/BRND-3110
